Mathlib/Topology/CWComplex/Classical/Basic.lean

```diff
@@ -222,6 +222,34 @@ lemma RelCWComplex.map_zero_mem_closedCell [RelCWComplex C D] (n : ℕ) (i : cel
     map n i 0 ∈ closedCell n i :=
   openCell_subset_closedCell _ _ (map_zero_mem_openCell _ _)
 
+/-- This is an auxiliary lemma used to prove `RelCWComplex.eq_of_eq_union_iUnion`. -/
+private lemma RelCWComplex.subset_of_eq_union_iUnion [RelCWComplex C D] (I J : Π n, Set (cell C n))
+    (hIJ : D ∪ ⋃ (n : ℕ) (j : I n), openCell (C := C) n j =
+      D ∪ ⋃ (n : ℕ) (j : J n), openCell (C := C) n j) (n : ℕ) : I n ⊆ J n := by
```

Comment:
Can you e.g. add a line break after the colon, before the statement, please? It's too easy to overlook it right now.